### PR TITLE
Disable tooltips on mobile

### PR DESCRIPTION
**Pull request by cjdenio**
Mostly fixes the issue addressed in #2396. Also makes it so tooltips don't pop up after tapping something on mobile.

The `static_pages#index` change fixes this related bug:

https://user-images.githubusercontent.com/34525547/167944124-bafbed61-c355-4154-bcca-10a9ae747d91.mov

### CODE REVIEW

File: app/assets/stylesheets/components/_tooltips.scss [hunk 1]
```diff
@@ -1,67 +1,71 @@
...
-    pointer-events: none;
-    position: absolute;
-    right: 100%;
-    text-align: center;
-    transform: translateY(50%);
-    transition: 0.125s all ease-in-out;
-    width: max-content;
-    z-index: 1;
-  }
+@media (min-width: 56em) {
```
Comment (maxwofford) [inline app/assets/stylesheets/components/_tooltips.scss]: Is there a variable we can be using here from the rest of our design system?

**Review by maxwofford — APPROVED** [review]
Added a comment, but happy with this going in if it works locally. Once it's merged, please reply to https://hackclub.slack.com/archives/CN523HLKW/p1652292057605019 & check if it's working for users there.

File: app/assets/stylesheets/components/_tooltips.scss (hunk 1 above)
Comment (cjdenio) [reply app/assets/stylesheets/components/_tooltips.scss]: No, it's hard-coded in a few places

Screen Shot 2022-05-12 at 11 00 45 AM

Comment (maxwofford) [reply app/assets/stylesheets/components/_tooltips.scss]: lmao, sounds good

**Pull request merged**